src/_posts/2019-12-31-test-driving-a-cdn.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+asset-type: post
+name: test-driving-a-cdn
+title: Test Driving a CDN
+date: 2019-12-31 10:00:00 +00:00
+author: Christopher Eyre
+image:
+    src: /assets/custom/img/blog/swan.png
+tags:
+    - varnish
+    - CDN
+    - TDD
+abstract: How to test drive a CDN
+alias: [/2019/12/31/test-driving-a-cdn]
+---
+
+# How to test drive a Content Delivery Network
```

Comment:
The heading already appears at the top of the page, so I'd get rid of this (and perhaps put "How to test drive a Content Delivery Network" into the title attribute in the meta). This seems to be how most blog posts work; the "Private Jenkins Shared Libraries" that is referenced in the readme as a good standard example doesn't have a main title in the doc, and only uses sub-headings.

Reply:
Applied